### PR TITLE
Add utils for bech32 format address

### PR DESCRIPTION
**Pull request by rlan35**
We will migrate to bech32 as our address format for following reasons:

1. Avoid potential confusion with Ethereum's address.
2. bech32 allows a prefix which can distinguish between blockchains and main/testnet.
3. It's more human readable and easy to write down (no upper cases)

This PR is the basic util for bech32 address. We will migrate to it before mainnet v0, which means we need to ask our foundational node generate new addresses before that.

### CODE REVIEW

**Comment by rlan35** [comment]
Jenkins, test this please.

**Comment by rlan35** [comment]
Jenkins, test this please.

**Comment by rlan35** [comment]
Jenkins, test this please.

**Comment by rlan35** [comment]
finally passed.

File: internal/common/address_test.go [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+	"strings"
+	"testing"
+)
+
+func TestIsBech32Address(t *testing.T) {
+	tests := []struct {
+		str string
+		exp bool
+	}{
+		{"one1ay37rp2pc3kjarg7a322vu3sa8j9puahg679z3", true},
```
Comment (LeoHChen) [inline internal/common/address_test.go]: how about test cases like ne1, one2, 1one, 1one1, etc?

File: internal/utils/bytes_test.go [hunk 2]
```diff
@@ -64,7 +64,7 @@ func TestIsHex(t *testing.T) {
 		{"0xHELLO_MY_NAME_IS_STEVEN_@#$^&*", false},
```
Comment (harmony-ek) [inline internal/utils/bytes_test.go]: …?! LOL

File: internal/common/address_test.go (hunk 1 above)
Comment (rlan35) [reply internal/common/address_test.go]: Added those.

**Pull request merged**